modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/editor/BaseEditor.js

```diff
@@ -131,6 +131,10 @@ const BaseEditor = forwardRef(
 					return contents;
 				},
 			};
+
+			return () => {
+				window[name] = null;
+			};
```

Comment:
Can we do `delete window[name];` here? Do we need to keep information that editor with this name was previously created?

Reply:
+1 to this request

Reply:
Sure! changed